tools/internal_ci/linux/grpc_xds_k8s.sh

```diff
@@ -45,6 +45,9 @@ build_test_app_docker_images() {
   gcloud -q auth configure-docker
   docker push "${CLIENT_IMAGE_NAME}:${GIT_COMMIT}"
   docker push "${SERVER_IMAGE_NAME}:${GIT_COMMIT}"
+  branch_name=$(echo "$KOKORO_JOB_NAME" | sed -E 's|^grpc/core/(.+)/linux/grpc_xds_k8s$|\1|')
+  tag_and_push_docker_image "${CLIENT_IMAGE_NAME}" "${GIT_COMMIT}" "${branch_name}"
+  tag_and_push_docker_image "${SERVER_IMAGE_NAME}" "${GIT_COMMIT}" "${branch_name}"
```

Comment:
Since this var is only available in kokoro, I'd recommend checking if it present here, instead of adding kokoro-specific logic (INITIATOR) to generic-sounding `tag_and_push_docker_image`

```suggestion
if [[ -n $KOKORO_JOB_NAME ]]; then
  tag_and_push_docker_image "${CLIENT_IMAGE_NAME}" "${GIT_COMMIT}" "${branch_name}"
  tag_and_push_docker_image "${SERVER_IMAGE_NAME}" "${GIT_COMMIT}" "${branch_name}"
fi
```

Reply:
Added the variable check for cpp/python.